docs/core/install/linux-rhel.md

```diff
@@ -101,13 +124,13 @@ The .NET Core Runtime allows you to run apps that were made with .NET Core that
 ```bash
 subscription-manager repos --enable=rhel-7-server-dotnet-rpms
 yum install rh-dotnet31-aspnetcore-runtime-3.1 -y
-scl enable rh-dotnet31-aspnetcore-runtime-3.1 bash
+scl enable rh-dotnet31 bash
```

Comment:
This was a mistake in the existing doc. The `scl` command takes the collection name, which is `rh-dotnet31`.